draft-packages/form/KaizenDraft/Form/Primitives/InputSearch/InputSearch.module.scss

```diff
@@ -8,409 +8,246 @@
 $input-height: 48px;
 $input-base-padding-horizontal: $spacing-sm;
 $input-icon-size: 1.25rem; // 20px
+// @note: had *2 at first, but the gap looked too big compared to UI Kit
```

Comment:
Please make a ticket for this so design can be made aware of this magic number

Reply:
Oops I forgot to remove the note 😂 It was just notes for me :')

But I certainly want design to take a look at before merging anyway (thus why I tagged them in Slack).